newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationPreview/index.js

```diff
@@ -104,19 +117,30 @@ export default class AnimationPreview extends Component<Props, State> {
           resourcesLoader={resourcesLoader}
           project={project}
         />
-        <Line>
-          <Column expand>
-            <FlatButton
-              icon={<Replay />}
-              label="Replay"
-              onClick={this.replay}
-            />
-            <FlatButton
-              icon={paused ? <PlayArrow /> : <Pause />}
-              label={paused ? 'Play' : 'Pause'}
-              onClick={paused ? this.play : this.pause}
-            />
-          </Column>
+        <Line noMargin alignItems="center">
+          <Timer style={styles.timeIcon} />
+          <TextField
+            value={timeBetweenFrames}
+            onChange={(e, text) => {onChangeTimeBetweenFrames(text); this.replay()}}
+            id="direction-time-between-frames"
+            type="number"
+            step={0.01}
+            precision={1}
+            min={0.01}
+            max={5}
```

Comment:
Should we allow bigger values? Like up to 10 or 20 seconds? Maybe even 60? Won't be used a lot but at least people can put it.